src/components/people.tsx

```diff
@@ -0,0 +1,177 @@
+/* eslint-disable max-len */
+import React, { useContext, useEffect, useState } from 'react';
+import { Link, useSearchParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { MyContext } from './state';
+import { Loader } from './Loader';
+import { Error } from './error';
+import { Person } from './person';
+import { Person as PersonType } from '../types';
+import { PeopleFilters } from './PeopleFilters';
+
+export const People = () => {
+  const { person, setPerson, error, setError, setIsLoading, isLoading } =
+    useContext(MyContext);
```

Comment:
It is not a good practice to destructure all the state variables and methods from the context in one line. This can lead to naming conflicts and also makes it difficult to track the source of these variables and methods. It would be better to use the context directly or rename variables and methods to indicate that they are coming from the context.